chef/cookbooks/ohai/files/default/plugins/linux/block_device_extended.rb

```diff
@@ -25,6 +25,21 @@
         File.open("/sys/block/#{dir}/queue/#{check}") { |f| block[dir][check] = f.read_nonblock(1024).strip }
       end
     end
+
+    # correct that the aacraid driver always sets the removable flag
+    if block[dir]["removable"] == "1"
+      path = File.readlink("/sys/block/#{dir}").split("/")
```

Comment:
Can't you use `block_device_dir` value here?

Reply:
I could, but I wanted to keep within the style of the rest.